react/components/AvailableFilters.js

```diff
@@ -10,8 +10,11 @@ const AvailableFilters = ({
   preventRouteChange = false,
   initiallyCollapsed = false,
   navigateToFacet,
-}) =>
-  filters.map(filter => {
+  keepOneFilterOpened = false,
+}) => {
+  const [open, setOpen] = useState({})
```

Comment:
Do you think that using `useContext` would work here instead of `useState`?

Reply:
I think useState is more appropriate here